python-for-android/dists/kolibri/src/main/java/org/learningequality/NotificationBuilder.java

```diff
@@ -0,0 +1,60 @@
+package org.learningequality;
+
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.provider.Settings;
+
+import androidx.core.app.NotificationCompat;
+
+import org.learningequality.Kolibri.R;
+
+public class NotificationBuilder extends NotificationCompat.Builder {
+    public NotificationBuilder(Context context, String channelId) {
+        super(context, channelId);
+        setSmallIcon(R.drawable.ic_stat_kolibri_notification);
+        setPriority(NotificationCompat.PRIORITY_LOW);
+        setColor(context.getColor(R.color.primary));
+        setSilent(true);
+
+        // Default title
+        String notificationTitle = context.getApplicationContext().getString(R.string.app_name);
+        setContentTitle(notificationTitle);
+
+        // defaults for service notification channel
+        if (channelId.equals(NotificationRef.ID_CHANNEL_SERVICE)) {
+            setOngoing(true);
```

Comment:
Yeah, looking at the docs for it, it does seem like it makes the notification more rather than less prominent: https://developer.android.com/reference/android/app/Notification.Builder#setOngoing(boolean)